crates/hyperswitch_interfaces/Cargo.toml

```diff
@@ -25,6 +26,9 @@ thiserror = "1.0.58"
 time = "0.3.35"
 
 # First party crates
+actix-web = "4.5.1"
+api_models = { version = "0.1.0", path = "../api_models" }
```

Comment:
Please try to remove dependency for api_models

Reply:
`api_models` has `IncomingWebhookDetails`, `ObjectReferenceId` and `ConnectorWebhookSecrets` hence this dependency cannot be removed.